pkg/oc/cli/cancelbuild/cancelbuild.go

```diff
@@ -244,9 +244,18 @@ func (o *CancelBuildOptions) RunCancelBuild() error {
 				o.ReportError(fmt.Errorf("build %s/%s failed to update: %v", build.Namespace, build.Name, err))
 				return
 			}
+			statusMsg := &printers.NamePrinter{Operation: "marked for cancellation, waiting to be cancelled"}
+			statusMsg.PrintObj(kcmdutil.AsDefaultVersionedOrOriginal(build, nil), o.Out)
 
 			// Make sure the build phase is really cancelled.
-			err = wait.Poll(500*time.Millisecond, o.timeout, func() (bool, error) {
+			timeout := o.timeout
+			if build.Spec.Strategy.JenkinsPipelineStrategy != nil {
+				//bump the timeout in case we have to wait for Jenkins
+				//to come up so that the sync plugin can actually change
+				//the phase
+				timeout = timeout + (90 * time.Second)
```

Comment:
i'd go at least 3 mins.  jenkins startup is slooooow.

Reply:
increase pushed